tox.ini

```
@@ -25,12 +25,32 @@ cmake-options =
     -DCMAKE_BUILD_TYPE=Debug \
     -DMOPS_TESTS=ON
 
-
 commands =
     cmake {toxinidir}/mops -B {envdir}/build {[testenv:cxx-tests]cmake-options}
     cmake --build {envdir}/build --config Debug
     ctest --test-dir {envdir}/build --build-config Debug --output-on-failure
 
+[testenv:build-python]
+# this environement makes sure one can build sdist and wheels for Python
+deps =
+    setuptools
+    wheel
+    cmake
+    twine
+
+allowlist_externals =
+    bash
+
+commands =
+    # check building sdist and wheels from a checkout
+    python setup.py sdist
+    python setup.py bdist_wheel
```

Let's use `build` here
```suggestion
    python -m build .  --outdir dist
```

This will build the sdist, and then try to build the wheel from the sdist